src/datachain/lib/dc.py

```diff
@@ -1069,13 +1069,22 @@ def mutate(self, **kwargs) -> "Self":
            Filename:    name(), parent(), file_stem(), file_ext()
            Array:       length(), sip_hash_64(), euclidean_distance(),
                         cosine_distance()
+           Window:      row_number(), rank(), dense_rank()
```

Comment:
add `first`?

Reply:
ah, and I see that we support actually a whole set of functions?

Reply:
Updated!
I don't think we support a whole set of functions, it is differs for SQLite and ClickHouse and other databases.

Please, let me know if I need to add more functions, I also can do it in the follow-up PRs.

Reply:
what is the usual set of functions? do they all support sum, avg - that type of things? we can create a followup ticket for this I think. Thanks!

Reply:
Created a follow-up ticket for that: https://github.com/iterative/datachain/issues/523